### PR TITLE
docs: Updated using git doc to help people with existing setups

### PR DESCRIPTION
**Pull request by laf**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

### CODE REVIEW

File: doc/Developing/Using-Git.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ git clone git@github.com:/yourusername/librenms.git
 ```
 
 If you are starting with an existing LibreNMS install either from our Virtual machine or install docs then you 
-will need to update the remote url. To do that run:
+will need to update the remote url. Be aware that daily.sh will not work properly after changing your origin. To do that run:
```
Comment (laf) [inline doc/Developing/Using-Git.md]: What do you see happening here?

It may just need `git branch --set-upstream-to=origin/master master` setting?

Comment (murrant) [inline doc/Developing/Using-Git.md]: You need that and you need to merge upstream manually to update.

Comment (laf) [inline doc/Developing/Using-Git.md]: Are you sure? I tested mine doing the set-upstream and can switch between urls and run daily.sh with no issue.

If you're correct then I'll close this as we shouldn't break daily.sh and expect people to fix it themselves.

Comment (murrant) [inline doc/Developing/Using-Git.md]: ./daily.sh works, but it only synchronizes with your fork... the current origin.

Comment (laf) [inline doc/Developing/Using-Git.md]: I'll close :)

**Pull request closed without merging**